### PR TITLE
G branch

### PR DESCRIPTION
**Pull request by GeorgeDelSims**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -129,6 +129,12 @@ dmypy.json
 .pyre/
 
 data
+
+#Test docs
+tests_G.ipynb
```
Comment (naimabe) [inline .gitignore]:
```suggestion

```

File: Main/model.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 from keras.models import Sequential, Model
 from keras import layers
 import tensorflow as tf
+from keras.callbacks import EarlyStopping
```
Comment (naimabe) [inline Main/model.py]: sort import

File: Main/model.py [hunk 3]
```diff
@@ -26,6 +27,16 @@ def initialise_EfficientNet_model():
...
 
 
+def train_model(model, X, y, batch_size=64, patience=5, validation_split=0.3):
+    '''
+    Train model with earlystopping and batch size parameters
+    Args:
+
+    KwArgs: 
+
+    '''
```
Comment (naimabe) [inline Main/model.py]: improve docs

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# setup.py
+from setuptools import setup
+
+setup(name='melanoma-prediction',
+        description="package description",
+        packages=["melanoma-prediction"]) # You can have several packages, try it
```
Comment (naimabe) [inline setup.py]:
```suggestion
        packages=["src"]) # You can have several packages, try it
```

File: Main/model.py (hunk 2 above)
Comment (GeorgeDelSims) [reply Main/model.py]: thanks I will do it

**Review by naimabe — APPROVED** [review]

**Pull request merged**